texttospeech/ssml_addresses/tts.py

```diff
@@ -0,0 +1,143 @@
+        return
+
+    # Instantiates a client
+    client = texttospeech.TextToSpeechClient()
+
+    # Sets the text input to be synthesized
+    synthesis_input = texttospeech.types.SynthesisInput(ssml=ssml_text)
+
+    # Builds the voice request, selects the language code ("en-US") and
+    # the ssml voice gender ("MALE")
+    voice = texttospeech.types.VoiceSelectionParams(
+        language_code='en-US',
+        ssml_gender=texttospeech.enums.SsmlVoiceGender.MALE)
+
+    # Selects the type of audio file to return
+    audio_config = texttospeech.types.AudioConfig(
+        audio_encoding=texttospeech.enums.AudioEncoding.MP3)
+
+    # Performs the text-to-speech request on the text input with the selected
+    # voice parameters and audio file type
+    response = client.synthesize_speech(synthesis_input, voice, audio_config)
+
+    # Writes the synthetic audio to the output file.
+    with open(outfile, 'wb') as out:
+        out.write(response.audio_content)
+        print('Audio content written to file ' + outfile)
+    # [END tts_ssml_address_audio]
+
+
+# [START tts_ssml_address_ssml]
+def text_to_ssml(inputfile):
+    # Generates SSML text from plaintext.
+    # Given an input filename, this function converts the contents of the text
+    # file into a string of formatted SSML text. This function formats the SSML
+    # string so that, when synthesized, the synthetic audio will pause for two
+    # seconds between each line of the text file. This function also handles
+    # special text characters which might interfere with SSML commands.
+
+    # Args:
+    # inputfile: string name of plaintext file
+
+    # Returns:
+    # A string of SSML text based on plaintext input
+
+    # Parse lines of input file
+    try:
+        open_file = open(inputfile, 'r')
+        raw_lines = open_file.readlines()
+
+    # Checks to make sure that the input file exists
+    except IOError:
+        print('text_to_ssml() error. File ' + inputfile + ' does not exist.')
+        return
+
+    # Define SSML timed break between addresses
+    time = '"2s"'
+    brk = '<break time=' + time + '/>'
+
+    # Initialize SSML script
+    ssml = '<speak>'
+
+    # Iterate through lines of file
+    for line in raw_lines:
+
+        # Replace special characters with HTML Ampersand Character Codes
+        # Some special characters might interfere with the SSML interpreter
+        line = line.replace('&', '&amp;')
+        line = line.replace('<', '&lt;')
+        line = line.replace('>', '&gt;')
+
+        # Concatenate the line to the SSML script
+        ssml += line
+
+        # Wait 2 seconds between speaking each address
+        ssml += brk
+
+    # Terminate the SSML script
+    ssml += '</speak>'
+
+    # Return the concatenated string of ssml script
+    return ssml
+# [END tts_ssml_address_ssml]
+
+
+# [START tts_ssml_address_test]
+if __name__ == '__main__':
+    # test example address file
+    plaintext = 'resources/example.txt'
```

Comment:
Running your script as is, creates a fairly strange audio file, because of the included region tags. 
I'd suggest removing the regions tags from any files in the `resources` folder.